Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/Shadow/DirectionalLightShadowCalculator.azsli

```diff
@@ -149,19 +155,21 @@ void DirectionalShadowCalculator::ComputeShadowCoords()
             m_cascadeIndex = index;
         }
     }
-            
+
+// Todo: No ddx in compute, find a way to leverage world space normal for this
```

Comment:
FYI we will have this when we upgrade DXC and enable SM6.6: https://microsoft.github.io/DirectX-Specs/d3d/HLSL_SM_6_6_Derivatives.html#derivative-functions